code/_core/datum/damagetype/ranged/bullet/rifle_223.dm

```diff
@@ -63,12 +63,14 @@
 
 	//The base attack damage of the weapon. It's a flat value, unaffected by any skills or attributes.
 	attack_damage_base = list(
-		PIERCE = DAMAGE_SWORD
+		PIERCE = DAMAGE_SWORD,
+		BIO = DAMAGE_SWORD*0.5
```

Comment:
The damage should be equal to one whole of the define. Right now this is equal to 1.5 parts, which isn't up to standards. You should upgrade the damage type to ``DAMAGE_AXE`` and use ``PIERCE = DAMAGE_AXE*0.5`` and ``BIO = DAMAGE_AXE*0.5``.

But I'm not sure I'm comfortable with a regular bullet dealing bio damage. While lead poisoning exists, it doesn't make sense that this bullet has it and other bullets don't.

Reply:
Fair enough. Aye, I'll change it.